Resources/Prototypes/_RMC14/Catalog/Fills/Crates/weapon_specialist_ammo.yml

```diff
@@ -52,6 +52,7 @@
     - id: RMCRocket84mm
       amount: 2
     - id: RMCRocket84mmAntiArmor
+      amount: 2
```

Comment:
You were right about the crate price being off, it's supposed to be $3000 with one of each rocket, adding up to three rockets. Change it in this file:
`Resources/Prototypes/_RMC14/Catalog/Cargo/requisitions_catalog.yml` 
```suggestion
      amount: 1
```
![image](https://github.com/user-attachments/assets/895ab431-5603-4270-b923-6803529acbc0)